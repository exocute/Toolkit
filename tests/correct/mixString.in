Graph MixString
	Import String
	Export String
	Activity A FilterCommas
	Import String
	Export String
	Activity B RemoveFirst
	Import String
	Export String
	Activity C RemoveLast
	Import String
	Export String
	Activity D JoinStrings
	Import String
	Export String
	
	Connection [A;B,C] [B;D] [C;D]