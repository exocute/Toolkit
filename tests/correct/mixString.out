Name: MixString
ImportName: String
ExportName: String
Id: D, Name: JoinStrings, Import: String, Export: String
Id: A, Name: FilterCommas, Import: String, Export: String
Id: C, Name: RemoveLast, Import: String, Export: String
Id: B, Name: RemoveFirst, Import: String, Export: String
A:B,C
C:D
B:D